buildspec.yaml

```diff
@@ -0,0 +1,14 @@
+version: 0.1
+
+phases:
+  install:
+    commands:
+      - echo Build started on `date`
+      - echo Installing required libs...
+      - npm install
```

Comment:
'yarn' may be better as it installs dependencies faster. not sure if that would be an issue to use instead?